master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Configuring calicoctl - etcdv2 datastore 
-layout: docwithnav
+title: Using calicoctl With an etcdv2 Datastore
```

Comment:
What about the equivalent title for the Kubernetes backend?  They should have parallel structure, so if you're changing the title of this one make sure you're also doing it there :)